api/signup.go

```diff
@@ -84,6 +88,10 @@ func (a *API) signupNewUser(ctx context.Context, params *SignupParams, aud strin
 	instanceID := getInstanceID(ctx)
 	config := a.getConfig(ctx)
 
+	if a.config.EmailBlacklist.Enabled && a.isEmailBlacklisted(params.Email) {
```

Comment:
personally, instead of using an `enabled` check here, I would just have a no-op blacklist implementation. It would just return false for everything. Or you could at least push this into the function `isEmailBlacklisted`

Reply:
👍